Neos.Fusion/Classes/FusionObjects/TagImplementation.php

```diff
@@ -77,6 +77,42 @@ public function evaluate()
         if (!$omitClosingTag && !$selfClosingTag) {
             $content = $this->fusionValue('content');
         }
-        return '<' . $tagName . $this->fusionValue('attributes') . ($selfClosingTag ? ' /' : '') . '>' . (!$omitClosingTag && !$selfClosingTag ? $content . '</' . $tagName . '>' : '');
+        $attributes = $this->fusionValue('attributes');
+        if (is_array($attributes)) {
+            $renderedAttributes = self::renderAttributes($attributes, true);
+        } else {
+            $renderedAttributes = (string)$attributes;
+        }
+        return '<' . $tagName . $renderedAttributes . ($selfClosingTag ? ' /' : '') . '>' . (!$omitClosingTag && !$selfClosingTag ? $content . '</' . $tagName . '>' : '');
+    }
+
+    /**
+     * @param array $attributes
+     * @param bool $allowEmpty
+     */
+    public static function renderAttributes(array $attributes, $allowEmpty = true): string
+    {
+        $renderedAttributes = '';
+        foreach ($attributes as $attributeName => $attributeValue) {
+            $encodedAttributeName = htmlspecialchars($attributeName, ENT_COMPAT, 'UTF-8', false);
+            if ($attributeValue === null || $attributeValue === false) {
+                // No op
```

Comment:
Nitpick: While you're at it.. This could be moved one line above and changed to
```php
if ($attributeValue === null || $attributeValue === false) {
    continue;
}
```

Reply:
Ack, is adjusted